modules/tinymce/src/themes/silver/main/ts/ui/toolbar/button/ToolbarButtons.ts

```diff
@@ -60,7 +62,15 @@ interface ChoiceFetcher {
 
 const getButtonApi = (component: AlloyComponent): Toolbar.ToolbarButtonInstanceApi => ({
   isEnabled: () => !Disabling.isDisabled(component),
-  setEnabled: (state: boolean) => Disabling.set(component, !state)
+  setEnabled: (state: boolean) => Disabling.set(component, !state),
+  setText: (text: string) => {
```

Comment:
One here is a block the other one is not should the setText and setIcon be the same both just lambdas without block since they have just one single statement in them?

Reply:
Agreed. Done!